### ISSUE TITLE
Rule: SC4-1-2-aria-hidden-focus (PR#1166)

### ISSUE THREAD

**Issue by jkodu** [dequelabs/axe-core#1150]
Spec: https://auto-wcag.github.io/auto-wcag/rules/SC4-1-2-aria-hidden-focus.html

**Comment by jkodu** [comment]
PR merged. Closing.

**Comment by jkatherman** [comment]
This causes legacy modal implementations to fail where aria-hidden is used to make content outside a dialog inert, when it contains focusable items.  Are screen readers sufficiently supportive of aria-modal to allow this rule?
